### ISSUE TITLE
Fail-safe date and language visibility if High Contrast is set 

### ISSUE THREAD

**Issue by cH3n7i** [openHPI/xikolo-ios#682]
**Is your feature request related to a problem? Please describe.**
With several images being bright, release dates and language are not visible - let's make sure we take care of High Contrast. 

<img width="258" alt="Screenshot 2019-09-04 at 19 53 57" src="https://user-images.githubusercontent.com/6067406/64278961-44316f80-cf4e-11e9-9604-dcd8ada1f81c.png">

**Describe the solution you'd like**
When in High Contrast mode we should make could put a subtle background color to make sure there is a contrast. 

**Affected Component**
`CourseAreaListViewController`

**Comment by mathebox** [comment]
I totally agree! The gradient on the bottom of the image does not always work so well. But I also think that this problem is not restricted to the high contrast mode. We should make sure that the text is readable all the time.

To be honest, I don't have a good looking solution for this yet. The options I see are the following:
- Remove the date and language label from the `CourseCell`
  - Here the user loses essential information, doesn't he?
- Move the date and language label below the image
  - With this option, we would increase the height of the `CourseCell` even more.
- Move the label in a `UIVisualEffectView`
  - This might look a bit strange

Some things we check before moving on:
- How does Apple handle this issue in its image-based apps? What are other big players doing?
- Should the layout of the `CourseCell` differ for the usage on the dashboard and the course list?
- Does the Android app show the course date and language in the course list? @tobiasrohloff 
  - The web application does display these values in the course list

**Comment by tobiasrohloff** [comment]
A course card on Android looks like this:

![IMAGE 2019-09-10 17:37:53](https://user-images.githubusercontent.com/2922744/64628539-dd9ace80-d3f1-11e9-8a73-bb4de8dcb615.jpg)
